testing/test_dhcp.sh

```diff
@@ -92,17 +92,26 @@ cmd/run -b -s settle_sec=0 dhcp_lease_time=120s
 
 cat inst/result.log | sort | tee -a $TEST_RESULTS
 
-for iface in $(seq 1 5); do
+for iface in $(seq 1 6); do
     intf_mac=9a:02:57:1e:8f:0$iface
     ip_file=inst/run-9a02571e8f0$iface/scans/ip_triggers.txt
+    report_file=inst/run-9a02571e8f0$iface/nodes/ipaddr0$iface/tmp/report.txt
     cat $ip_file
     ip_triggers=$(fgrep done $ip_file | wc -l)
     long_triggers=$(fgrep long $ip_file | wc -l)
     num_ips=$(cat $ip_file | cut -d ' ' -f 1 | sort | uniq | wc -l)
+    ip_change=$(cat $report_file | fgrep 'pass connection.dhcp.ip_change' | wc -l)
+    dhcp_timeouts=$(cat inst/cmdrun.log | fgrep 'DHCP times out after 120s lease time' | wc -l)
     echo Found $ip_triggers ip triggers and $long_triggers long ip responses.
-    if [ $iface == 5 ]; then
+
+    if [ $iface == 6 ]; then
+      device_dhcp_timeouts=$(cat inst/cmdrun.log | fgrep 'DHCP times out after 120s lease time' | fgrep "ipaddr_ipaddr0$iface" | wc -l)
+      echo "Device $iface DHCP timeouts: $device_dhcp_timeouts" | tee -a $TEST_RESULTS
+      echo "Device $iface ip change: $((ip_change == false))" | tee -a $TEST_RESULTS
```

Comment:
That should be $ip_change and the output should be 0 to indicate there isn't an IP change to avoid confusions.